components/navbar.css

```diff
@@ -0,0 +1,49 @@
+.navbar,
+.navbar-title,
+.navbar-search,
+.navbar-icons {
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  justify-content: space-around;
+}
+
+.navbar {
+  background-color: var(--COLOR-PRIMARY);
+  color: var(--COLOR-FT);
+  padding: 0.5rem;
+}
+
+.navbar-search {
+  border-radius: 10px;
+  background: var(--COLOR-FT);
+  padding-left: 0.5rem;
+}
+
+.navbar-input {
+  border: none;
+  padding: 0.5rem;
+  outline: none;
+  padding-left: 0.5rem;
+  padding-right: 2rem;
```

Comment:
- for padding you can use shorthand property, the above code can be written as padding: 0.5rem 2rem 0.5rem 0.5rem.
- Also, instead of having whole other padding for the right side, you would have given **margin-right: auto** to the search icon, as you have already set .navbar-search as flex. Do try it.

Reply:
fixed